src/features/content-insights/ContentInsightsSummary.tsx

```diff
@@ -9,17 +10,24 @@ import './ContentInsightsSummary.scss';
 
 interface Props {
     graphData: GraphData;
+    isError: boolean;
```

Comment:
I'm wondering if this should be an object because we'll need to differentiate the permission error vs API error scenarios

Reply:
Good point. We use AxiosError in EUA, should we try and use the same typings maybe? Or we can be less explicit and use just an object for now.

Reply:
Yeah seems like `AxiosError` exists in BUIE as well so let's go with that for now